### DISCUSSION TITLE
Backing up Teslamate data to Onedrive

### DISCUSSION THREAD

**Discussion by bahree** [teslamate-org/teslamate#4041] [Show and tell]
I have been backing up my Teslamate backup from my Azure Ubuntu VM to onedrive and finally got around to blogging about it incase others are interested too.

Details here: https://blog.desigeek.com/post/2024/07/backing-teslamate-to-onedrive/

**Comment by danielpopovici745** [comment]
Thanks I will give it a shot

**Reply by danielpopovici745** [reply]
I was able to follow your blog and it helped alot. I ended up using docker-rclone since its a lot easier. The bash script was very helpful!
